autopr/integrations/github_app/callback.py

```
@@ -0,0 +1,75 @@
+
+from fastapi import APIRouter, HTTPException, Query
+from fastapi.responses import RedirectResponse
+from github import GithubIntegration
+
+callback_router = APIRouter(prefix="/api/github-app", tags=["github-app"])
+
+
+@callback_router.get("/callback")
+async def callback(
+    code: str | None = Query(None),
+    state: str | None = Query(None),
```

Ruff (0.14.7)

Unused function argument: `state`

(ARG001)